google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     compile ('com.google.cloud.tools:appengine-plugins-core:' + toolsLibVersion) {
         exclude group: 'com.google.guava', module: 'guava'
     }
+    compile 'org.yaml:snakeyaml:1.18'
```

Comment:
Take a look at AppEngineStandardSupportProviderTest. It requires javaee stuff, because it is testing bits of AppEngineStandardUltimateWebIntegration, and it extends JavaeeFrameworkSupportProviderTestCase too.